formats/mainstream_browse_page/frontend/examples/top_level_page.json

```diff
@@ -1,7 +1,9 @@
 {
     "base_path": "/browse/benefits",
     "description": "Includes tax credits, eligibility and appeals",
-    "details": {},
+    "details": {
+        "child_ordering": "curated"
+      },
```

Comment:
It would be better to add another example with curated ordering, and leave this one for the common case.